src/main/resources/assets/iris/lang/de_de.json

```diff
@@ -2,36 +2,42 @@
   "iris.shaders.reloaded": "Shader neu geladen!",
   "iris.shaders.toggled": "Shader umgeschaltet auf: %s",
   "iris.keybind.reload": "Shader neu laden",
-  "iris.keybind.shaderPackSelection": "Shaderpaket-Auswahlbildschirm",
+  "iris.keybind.shaderPackSelection": "Shader-Auswahlbildschirm",
   "iris.keybind.toggleShaders": "Shader umschalten",
   "iris.keybinds": "Iris",
-  "iris.shaders.reloaded.failure": "Neuladen der Shader fehlgeschlagen! Grund: %s",
-  "iris.shaders.toggled.failure": "Umschalten der Shader fehlgeschlagen! Grund: %s",
+  "iris.shaders.reloaded.failure": "Shader konnte nicht neu geladen werden! Grund: %s",
+  "iris.shaders.toggled.failure": "Shader konnte nicht umgeschaltet werden! Grund: %s",
   "iris.sodium.failure.title": "Iris konnte nicht geladen werden!",
   "iris.sodium.failure.download": "Sodium herunterladen",
   "iris.sodium.failure.reason.notFound": "Iris erfordert Sodium für optimale Leistung, aber Sodium ist nicht installiert. Bitte lade Sodium herunter, füge es deinem Mods-Ordner hinzu und starte das Spiel neu.",
   "iris.sodium.failure.reason.incompatible": "Iris erfordert Sodium für optimale Leistung, aber es wurde eine inkompatible Version gefunden. Bitte lade die passende Version von Sodium herunter, füge sie deinem Mods-Ordner hinzu, entferne die falsche Version und starte das Spiel neu.",
 
   "options.iris.apply": "Anwenden",
   "options.iris.refresh": "Aktualisieren",
-  "options.iris.openShaderPackFolder": "Shaderpaket-Ordner öffnen ...",
-  "options.iris.shaderPackSettings": "Shaderpaket-Einstellungen ...",
-  "options.iris.shaderPackList": "Shaderpaket-Liste ...",
-  "options.iris.refreshShaderPacks": "Shaderpakete aktualisieren",
+  "options.iris.openShaderPackFolder": "Shader-Ordner öffnen ...",
+  "options.iris.shaderPackSettings": "Shader-Einstellungen ...",
+  "options.iris.shaderPackList": "Shader-Liste ...",
+  "options.iris.refreshShaderPacks": "Shader/Liste aktualisieren",
```

Comment:
In the English version this also doesn't say "Liste". What is your reasoning here?

Reply:
Pressing the button will also refresh the list to include shaderpacks that were manually added without drag and-drop and I wanted to imply that

Reply:
That makes sense. I guess the English version kind implies that the list is being refreshed without saying it. This way is good too.

Reply:
I just want to make it a little easier to understand what it actually does because “aktualisieren” also means update and users may press the button to update their shaderpacks then. This way that will less likely be the case